test/Graphics/Image.test.cpp

```diff
@@ -405,7 +419,8 @@ TEST_CASE("[Graphics] sf::Image")
 
         SECTION("Successful save")
         {
-            auto filename = std::filesystem::temp_directory_path();
+            auto                    filename  = std::filesystem::temp_directory_path();
+            std::array<char32_t, 6> utf32name = {'X', '.', 'p', 'n', 'g', '\0'};
```

Comment:
Another place we can use string literals instead. The comments are still great to explain what the hex codes mean so keep those please.